src/Microsoft.AspNet.Mvc.ViewFeatures/ViewFeatures/IHtmlGenerator.cs

```diff
@@ -303,7 +303,7 @@ public interface IHtmlGenerator
             string optionLabel,
             string expression,
             IEnumerable<SelectListItem> selectList,
-            IReadOnlyCollection<string> currentValues,
+            ICollection<string> currentValues,
```

Comment:
Already spoke to @Eilon about this minor correction in the API. Not a big deal since we expect few users to interact with an `IHtmlGenerator`.